packages/server/lib/makeDataContext.ts

```diff
@@ -62,9 +62,6 @@ export function makeDataContext (options: MakeDataContextOptions): DataContext {
       validateConfig: configUtils.validate,
```

Comment:
I will say, we are passing in `cypressVersion` above, but I'm not sure we need to. We are pulling `pkg.version` directly in the DataContext package in several places.

Reply:
Yeah, we can probably just do `require('@packages/root')` here